packages/truffle-interface-adapter/lib/web3-shim.js

```diff
@@ -3,30 +3,50 @@ const BN = require("bn.js");
 
 // This is a temporary shim to support the basic issues with Quorum
```

Comment:
(Risky move, calling a software component temporary!)

Reply:
I like to sign and date these sorts of comments so I can save someone the trouble of `git blame`ing me in the future.

Reply:
Well if this isn't temporary, then we need to revisit our roadmap and expectations for the next 4 months lol.

Git blaming takes a couple of minutes, but sure, I can date it.